openapi/components/schemas/Common/CommonPaymentCard.yaml

```
@@ -13,7 +13,7 @@ properties:
     enum:
       - payment-card
   customerId:
-    description: The сustomer's ID.
+    description: The customer's ID.
```

```suggestion
    description: The customer ID.
```

I usually ignore this, but since this line is changed, I think we can remove pointless `'s`

---

I recommend we make this change too.